app/models/requests/mediated_page.rb

```diff
@@ -40,10 +34,6 @@ def requestable_with_sunet_only?
     false
   end
 
-  def item_limit
-    ITEM_LIMITS[origin] || super
```

Comment:
In this context, super fell back to `nil`.  This PR effectively moves this logic up to the common viewer and since `ITEM_LIMITS['unconfigured-origin']` is also `nil` so the behavior is effectively identical (although we've sort-of removed a layer of indirection).